app/Http/Controllers/Admin/Workflow/BulkPublishingController.php

```diff
@@ -233,4 +233,62 @@ public function getBulkPublishStatus(): JsonResponse
             return response()->json(['success' => false, 'message' => 'Request error']);
         }
     }
+
+    /**
+     * Get status of bulk publishing for users belonging to same org.
+     *
+     * @return JsonResponse
+     *
+     * @throws \Exception
+     */
+    public function getOrganisationBulkPublishingStatus()
+    {
+        try {
+            $organizationId = auth()->user()->organization->id;
+
+            if ($organizationId) {
+                $publishStatus = $this->publishingStatusService->getActivityPublishingStatus($organizationId, '');
+                if (count($publishStatus)) {
+                    $response = $this->bulkPublishingService->getPublishingResponse($publishStatus);
+
+                    return response()->json(['success' => true, 'message' => $response['message'], 'data' => $response]);
+                }
+
+                return response()->json(['success' => true, 'message' => 'No bulk publishing in process.']);
+            }
+
+            return response()->json(['success' => false, 'message' => 'Insufficient data.']);
+        } catch (\Exception $e) {
+            logger()->error($e->getMessage());
+
+            return response()->json(['success' => false, 'message' => 'Status generation failed.']);
+        } catch (NotFoundExceptionInterface | ContainerExceptionInterface $e) {
+            logger()->error($e->getMessage());
+
+            return response()->json(['success' => false, 'message' => 'Request error']);
+        }
+    }
+
+    /**
+     * @param $organization
+     *
+     * @return JsonResponse
+     */
+    public function cancelBulkPublishing(): JsonResponse
+    {
+        try {
+            $this->bulkPublishingService->stopBulkPublishing(auth()->user()->organization->id);
```

Comment:
Check the success or failure condition. And based on that condition return JSON response